### PR TITLE
Revert mds bump to fix the isolated net 7 issue.

### PR DESCRIPTION
**Pull request by MaddyDev**
Due to MDS and (System.Runtime.Caching) package bump, we are seeing the issue #920 
Downgrading the packages to address that.

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -6,11 +6,11 @@
     <PackageVersion Include="Microsoft.Azure.Functions.Worker.Extensions.Abstractions" Version="1.2.0" />
     <PackageVersion Include="Microsoft.ApplicationInsights" Version="2.21.0" />
     <PackageVersion Include="Microsoft.Azure.WebJobs" Version="3.0.37" />
-    <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.1.1" />
+    <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.0.1" />
     <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="1.1.1" />
     <PackageVersion Include="morelinq" Version="3.4.2" />
     <PackageVersion Include="System.Drawing.Common" Version="7.0.0" />
-    <PackageVersion Include="System.Runtime.Caching" Version="7.0.0" />
+    <PackageVersion Include="System.Runtime.Caching" Version="5.0.0" />
```
Comment (Charles-Gagnon) [inline Directory.Packages.props:13]: Can we just revert to 6.0.0 for this instead? MDS 5.1.1 says it just requires this >= 6.0.0 so it would be nice to keep the MDS update if possible.

https://www.nuget.org/packages/Microsoft.Data.SqlClient/5.1.1#dependencies-body-tab

And then we should figure out if we can make a test to catch this kind of failure in the future.

File: builds/azure-pipelines/build-release.yml [hunk 2]
```diff
@@ -28,7 +28,7 @@ variables:
   versionPatch: $[counter(variables['versionMajorMinor'], 0)] # This will reset when we bump minor version
   binariesVersion: '$(versionMajor).$(versionMinor).$(versionPatch)'
   nugetVersion: $[replace(replace('True', eq('${{ parameters.ReleaseType }}', 'full'), '$(binariesVersion)'),'True','$(binariesVersion)-preview')] # This will set the preview tag to the version based on the ReleaseType parameter.
-  supportedVersion: $[replace(replace('True', eq('${{ parameters.ReleaseType }}', 'full'), '$(versionMajor).*'),'True','$(versionMajor).*-*')] # This will set the expression to pull the preview vs full version based on the ReleaseType parameter.
+  supportedVersion: $[replace(replace('True', eq('${{ parameters.ReleaseType }}', 'full'), '$(binariesVersion)'),'True','$(binariesVersion)-preview')] # This will set the expression to pull the preview vs full version based on the ReleaseType parameter.
```
Comment (Charles-Gagnon) [inline builds/azure-pipelines/build-release.yml:31]: Undo these changes, the other PR will take care of this

**Review by Charles-Gagnon — APPROVED** [review]

**Review by lucyzhang929 — APPROVED** [review]
